src/System.Runtime.Handles/tests/System.Runtime.Handles.Tests.csproj

```diff
@@ -20,6 +20,7 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <OutputItemType>Content</OutputItemType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
```

Comment:
Do we know why we have to explicitly specify these targets in the reference? I would much prefer that the actual partial facade project do the right thing here. The more we have these partial facades that are referenced via a ProjectReference the more I think we need to have a solution some where in our targets for these so we don't have to remember and copy all this extra metadata to reference them. @mellinoe something to look into.